roles/adpicam/templates/base.cmd.j2

```diff
@@ -0,0 +1,28 @@
+
+dbLoadDatabase("{{ deploy_ioc_template_root_path }}/dbd/{{ deploy_ioc_executable }}.dbd")
+{{ deploy_ioc_executable }}_registerRecordDeviceDriver(pdbbase)
+
+# adpicam specific commands
+
+# Create a PICam driver
+# PICamConfig(const char *portName, IDType, IDValue, maxBuffers, size_t maxMemory, int priority, int stackSize)
+
+# This is for a
+PICamConfig("$(PORT)", 0, 0, 0, 0)
+
+asynSetTraceIOMask($(PORT), 0, 2)
+
+dbLoadRecords("$(ADCORE)/db/ADBase.template", "P=$(PREFIX),R=cam1:,PORT=$(PORT),ADDR=0,TIMEOUT=1")
+dbLoadRecords("$(ADPICAM)/db/PICam.template","P=$(PREFIX),R=cam1:,PORT=$(PORT),ADDR=0,TIMEOUT=1")
+
+# Create a standard arrays plugin, set it to get data from Driver.
+NDStdArraysConfigure("Image1", 3, 0, "$(PORT)", 0)
+dbLoadRecords("$(ADCORE)/db/NDStdArrays.template", "P=$(PREFIX),R=image1:,PORT=Image1,ADDR=0,TIMEOUT=1,TYPE=Int16,SIZE=16,FTVL=SHORT,NELEMENTS=20000000,NDARRAY_PORT=$(PORT)")
+
+# Load all other plugins using commonPlugins.cmd
+< $(ADCORE)/iocBoot/commonPlugins.cmd
+set_requestfile_path("$(ADPICAM)/PICamApp/Db")
```

Comment:
```suggestion
```
I believe this can be scrapped and is taken care of in the default role. @jwlodek to confirm.

Reply:
Can confirm. Although I am unsure about the Image1 plugin since I didn't see it in the ADCore common plugins here: https://github.com/areaDetector/ADCore/blob/master/iocBoot/EXAMPLE_commonPlugins.cmd